server/similarity_utils.py

```diff
@@ -40,9 +40,8 @@ def perform_sim_search(self, images, query_phrase, top_k=3):
         logits_per_image = outputs.logits_per_image
 
         top_indices, top_scores = self.sort_scores(logits_per_image, top_k)
-        top_images = [images[index] for index in top_indices]
```

Comment:
We don't need to compute this if we are returning URLs only.

Reply:
Sorry about this, my bad to fix things too quickly. Let me fix this tomorrow

Reply:
@sayakpaul 
do you mean we don't need `top_indices, top_scores = self.sort_scores(logits_per_image, top_k)`?
I guess we still need `top_indices` to refer which URLs to return, 
so I just removed `top_images = [images[index] for index in top_indices]` part.

what do you think?

Reply:
Yes, got it.